CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ in the naming of release branches.
 
 ### Added
 
+- Add `PlutusDebug` to `TransactionScriptFailure.ValidationFailedV{1|2}`: #3135
```

Comment:
It is also worth adding here that the `Show` instance for `TransactionScriptFailure` has been removed

Reply:
Noting here: Since the latest discussion we have decided to keep a dummy `Show` instance for `PlutusDebug`.